Libplanet.Tests/Serialization/Bencodex/BencodexFormatterTest.cs

```diff
@@ -109,12 +116,12 @@ protected Person(SerializationInfo info, StreamingContext context)
         {
             Name = info.GetString("Name");
             Age = (int)info.GetInt64("Age");
-            Names =
-                ((List<object>)info.GetValue("Names", typeof(List<object>)))
+            Names = info.GetValue<List<object>>("Names")
```

Comment:
Is this change intended?  The previous code looks rather proper.

Reply:
I thought `info.GetValue<List<object>>("Names")` is better than `((List<object>)info.GetValue("Names", typeof(List<object>)))`. Do you think previous one is better?

Reply:
I thought it is not like better or worse, but just it's unlikely to be compiled.  If the type checker passes I don't mind then.

Reply:
To elaborate, the changed version looks like trying to putting a `List<object>` value into `List<string>` property, which C#'s type checker does not allow AFAIK.

Reply:
There is more code below:
```csharp
.Select(s => (string)s)
.ToList();
```
So I think it will pass the type checker.